engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkModelJar.scala

```diff
@@ -3,31 +3,30 @@ package pl.touk.nussknacker.engine.management
 import java.io.{File, FileOutputStream}
 import java.net.URL
 import java.nio.file.Files
-import java.util.jar.{JarEntry, JarInputStream, JarOutputStream}
+import java.util.jar.{JarEntry, JarOutputStream}
 import com.typesafe.scalalogging.LazyLogging
 import org.apache.commons.io.{FileUtils, IOUtils}
-import pl.touk.nussknacker.engine.{BaseModelData, ModelData}
 
 import scala.util.Using
 
-class FlinkModelJar extends LazyLogging {
+class FlinkModelJar(modelUrls: List[URL], includeDropwizardLibsImplicitly: Boolean = true) extends LazyLogging {
 
-  // TODO: handle multiple models?
   private var modelFile: Option[File] = None
 
   // we want to have *different* file names for *different* model data (e.g. after rebuild etc.)
   // currently we just generate random file names
-  def buildJobJar(modelData: BaseModelData, includeDropwizardLibsImplicitly: Boolean = true): File = synchronized {
+  def buildJobJar(): File = synchronized {
```

Comment:
I think that it's time to change the name of this method. It doesn't build jar every time when it is invoked. Also I would change the name of the class. It is a kind of `FlinkModeljarProvider`

Reply:
Done